assets/default.game.yaml

```diff
@@ -1,5 +1,8 @@
 start_level: beach/beach.level.yaml
 camera_height: 448
+# Allowed x distance between players, as a fraction of the screen width.
+# This is also used for camera move, so be careful.
+max_players_x_pct_distance: 0.25
```

Comment:
What is pct intended to stand for in max_payers_x_pct_distance? Could name be simplified at all?

Reply:
It stood for `pct`, but I've renamed it to `fraction`, which is very obvious.